### ISSUE TITLE
Project Setup

### ISSUE THREAD

**Issue by helpmefindaname** [helpmefindaname/ner-eval-dashboard#1]
Create a basic python setup with a Dockerfile, CI-CD, testing with pytest, mypy, black, flake8, isort and so on.

Create basic interfaces  for
* predictors such as flair, transformers, spacy.
* Elements (tables/plots/examples) such as F1 score/precision/recall per class;loss per token for the largest n tokens;examples with the highest uncertainty and their prediction. Ensure that plots might be optional depending on the predictor. 
* Datasets being able to handle unlabeled sentences, a test set and the training & validation sets. 

Basic functionality to create a dashboard based on all Elements available for a given predictor and the data availability of the data set.
